include.js

```diff
@@ -2903,6 +2903,13 @@ let cards = [
     author: 'Kalliran',
     githubLink: 'https://github.com/Kalliran'
   },
+  {
+    artName: 'Animrotation', 
+    pageLink: './Art/danielbarrosdevs/index.html',
+    imageLink: './Art/danielbarrosdevs/Animrotation.gif', 
+    author: 'Daniel', 
+    githubLink: 'https://github.com/danielbarrosdevs' 
+  }
```

Comment:
# Paths not matching contribution 
`./Art/danielbarrosdevs` indicates your contribution lies within the folder "danielbarrosdevs" but your foldername is named "Animrotatation"
These path will need to be reviewed along with your folder name